src/App.js

```diff
@@ -14,14 +17,23 @@ const useStyles = makeStyles(() => ({
 export default function App() {
   const classes = useStyles();
 
+  const map = new OLMap({
+    controls: [],
+    interactions: defaultInteractions({
+      altShiftDragRotate: false,
+      pinchRotate: false,
+    }),
+  });
+
   return (
     <Grid container className={classes.container}>
       <Grid item xs={6} className={classes.mapLeft}>
         <Map permalinkParam="left" />
       </Grid>
       <Grid item xs={6} className={classes.mapRight}>
-        <Map permalinkParam="right" />
+        <Map permalinkParam="right" map={map} />
       </Grid>
+      <Zoom map={map} />
```

Comment:
May be one Zoom for each map. Ask uli

Reply:
Uli asked for a zoom per map. I changed it